### ISSUE TITLE
New rule: disallow assertions with `and`

### ISSUE THREAD

**Issue by m-burst** [m-burst/flake8-pytest-style#12]
Bad:
```py
assert something and something_else
```

Good:
```py
assert something
assert something_else
```

Rationale: simplify assertions, check each result separately
